models/models.js

```diff
@@ -79,4 +79,15 @@ exports.updateArticleVotes = ({ inc_votes }, id) => {
             return rows[0]
         }
     })
+}
+
+exports.deleteSpecificComment = (id) => {
```

Comment:
this is being picky just because you're doing very well, but I would maybe call this something like deleteCommentById to be a bit more descriptive